src/Tizen.System.SystemSettings/Tizen.System.SystemSettings/SystemSettings.cs

```diff
@@ -1145,6 +1145,40 @@ public static bool AccessibilityNegativeColor
             }
         }
 
+        /// <summary>
+        /// Indicates whether rotary event is enabled on the device.
+        /// </summary>
+        /// <privilege>http://tizen.org/privilege/systemsettings.admin</privilege>
+        /// <privlevel>platform</privlevel>
+        /// <feature>http://tizen.org/feature/systemsetting</feature>
+        /// <feature>http://tizen.org/feature/accessibility.negative</feature>
+        /// <exception cref="ArgumentException">Invalid Argument</exception>
+        /// <exception cref="NotSupportedException">Not Supported feature</exception>
+        /// <exception cref="InvalidOperationException">Invalid operation</exception>
+        /// <exception cref="UnauthorizedAccessException">Thrown when application does not have privilege to access this method.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static bool RotaryEventEnabled
+        {
+            get
+            {
+                bool isRotaryEventEnabled;
+                SystemSettingsError res = (SystemSettingsError)Interop.Settings.SystemSettingsGetValueBool(SystemSettingsKeys.RotaryEventEnabled, out isRotaryEventEnabled);
+                if (res != SystemSettingsError.None)
+                {
+                    throw SystemSettingsExceptionFactory.CreateException(res, "unable to get isRotaryEventEnabled system setting.");
```

Comment:
X AVOID throwing exceptions from property getters.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/property

You can just return false as default value

Reply:
I need to exception. Because this class depends on system-settings native API, it need to be noticed error exception for user.

Reply:
Which case could be thrown exception?
And it need to notify to user?
Why need it?